terraform-tests/aurora_mysql_test.go

```diff
@@ -28,7 +28,7 @@ var _ = Describe("Aurora mysql", Label("aurora-mysql-terraform"), Ordered, func(
 		"cluster_instances":                      3,
 		"serverless_min_capacity":                nil,
 		"serverless_max_capacity":                nil,
-		"engine_version":                         nil,
+		"engine_version":                         "8.0",
```

Comment:
I think this is okay. Although we don't have a default in our brokerpak and it is not a required field we wouldn't expect customers to create instances without providing a specific version. However it would be good to document that it is not recommended to create an instance without specifying a version either in the plan or in the provision params. Not really pertinent to this code review, but we may want to create a doc story to play before/during the release.

Reply:
Thank you very much @pivotal-marcela-campo . I just created a story to play in the next days.